### PR TITLE
Rearrange command handlers

### PR DESCRIPTION
**Pull request by Pilatuz**
All command handlers are moved to dedicated header/source files.

### CODE REVIEW

File: firmware-src/sources/user_config.h [hunk 1]
```diff
@@ -36,4 +36,65 @@
...
+// customize command set (compile time)
+#define DH_COMMANDS_GPIO     // enable GPIO commands
+#define DH_COMMANDS_ADC      // enable ADC commands
+#define DH_COMMANDS_PWM      // enable PWM commands
+#define DH_COMMANDS_UART     // enable UART commands
+#define DH_COMMANDS_I2C      // enable I2C commands
+#define DH_COMMANDS_SPI      // enable SPI commands
+#define DH_COMMANDS_ONEWIRE  // enable onewire commands
+
+// TODO: customize device list
```
Comment (Nikolay-Kha) [inline firmware-src/sources/user_config.h:98]: have you already done it, haven't you?

File: DeviceHiveESP8266.md [hunk 2]
```diff
@@ -200,37 +204,59 @@ ssid=ssid&pass=pass&url=http%3A%2F%2Fexample.com%2Fapi&id=deviceid&key=accesskey
...
-      "14"            | GPIO14, SPI_CLK  |        9          |       D5
-      "15"            |    GPIO15        |        13         |       D8
-ADC                   |                  |                   |
-      "0"             |    ADC0          |        6          |       A0
-Common                |                  |                   |
-      "0"             |          all pins in current group
-  
-_Notes:  
-GPIO6-GPIO11 usually connected to on-module EEPROM, that is why no API for this pins._
+| Pin name in commands |  Function        |ESP8266 pin number |NodeMCU board pin |
```
Comment (Nikolay-Kha) [inline DeviceHiveESP8266.md:207]: Table looks broken. See here - https://github.com/Pilatuz/esp8266-firmware/blob/develop/DeviceHiveESP8266.md#pin-definition

**Review by Nikolay-Kha — COMMENTED** [review]
lgtm

File: firmware-src/sources/user_config.h (hunk 1 above)
Comment (Pilatuz) [reply firmware-src/sources/user_config.h:98]: no, I haven't

File: DeviceHiveESP8266.md (hunk 2 above)
Comment (Pilatuz) [reply DeviceHiveESP8266.md:207]: it was broken. take a look at code-review branch https://github.com/Pilatuz/esp8266-firmware/blob/code-review/DeviceHiveESP8266.md#pin-definition

**Pull request merged**